client/src/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -53,7 +53,7 @@ const RelatedProducts = ({ productId, handleClick, mainProductDetails, reviews }
         </ul>
       </div>
       <a href="#" className="jcarousel-control-prev disabled" data-jcarouselcontrol="true">‹</a>
-      <a href="#" className="jcarousel-control-next disabled" data-jcarouselcontrol="true">›</a>
+      <a href="#" className="jcarousel-control-next disabled" data-jcarouselcontrol="true">{console.log('Active: ', $('.inactive'))}›</a>
```

Comment:
Did you want to keep this console.log() in here? Also, were you planning to remove the "disabled" tag from this element as well?

Reply:
My apologies. I think my brain is slowly frying from looking at these arrows for so long. I will take care of these!